### PR TITLE
Demo scripts + Repl verbose flag

### PR DESCRIPTION
**Pull request by kwannoel**

### CODE REVIEW

**Comment by kwannoel** [comment]
Sample cmds @thomastanck 

```sh
yarn repl -v src/demo/functions.rkt
yarn repl src/demo/macros.rkt
```

File: src/demo/macros.rkt [hunk 1]
```diff
@@ -0,0 +1,44 @@
+(define add
+  (#%plain-lambda (a b)
+    (+ a b)))
+
+(add 1 2)
```
Comment (thomastanck) [inline src/demo/macros.rkt:5]: are you not console-log'ing this in the demo?

File: src/demo/macros.rkt [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                     (console-log func-name)
+                     (console-log args)
+                     (console-log body)
+                     (list 'define func-name
+                           (list '#%plain-lambda args body)))))
+
+(defun1 add_ (a b)
+  (+ a b))
+
+(add_ 1 2)
```
Comment (thomastanck) [inline src/demo/macros.rkt:22]: saem

**Review by thomastanck — COMMENTED** [review]
LGTM, though you might want to show off how our standard library makes both writing programs and writing macros easier. in particular looking at how `defun` could really benefit from the quasiquote and syntax-case libraries (not at the same time).

**Pull request merged**